dts/bindings/flash_controller/st,stm32-xspi-nor.yaml

```diff
@@ -51,3 +51,12 @@ properties:
     enum:
       - 1
       - 2
+
+  ncs-line:
+    type: int
+    default: 1
```

Comment:
required: true instead of default?

Reply:
The setting is not present for H5, so it shouldn't be required. I've added ifdef for it in the driver to prevent compilation errors.